modules/ROOT/pages/sap/sap-connector.adoc

```diff
@@ -25,23 +25,43 @@ Mule runtime supports SAP integration through our Anypoint Connector for SAP, an
 * Receive IDocs over tRFC and qRFC.
 * Transform SAP objects (JCo Function/BAPI & IDocs) to and from XML.
 
+== About Connectors
+
+Anypoint connectors are Mule runtime engine extensions that enable you to connect to APIs 
+and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twitter.
+
 == Prerequisites
 
+Before creating an app, you must have access to the SAP target resource and Anypoint Platform. You 
+must also understand how to create a Mule app using Anypoint Studio.
+
 This document assumes you have:
 
 * A working knowledge of the SAP business context and in particular, the SAP R/3 Business Suite.
 * A basic understanding of the SAP NetWeaver Platform from an administration point of view.
 * Knowledge of the ABAP language.
 * Login credentials to test your connection to your target resource.
 
-For hardware and software requirements and compatibility information, see the 
+For software requirements and compatibility information, see the 
 xref:release-notes::connector/sap-connector-release-notes-mule-4.adoc[SAP Connector Release Notes].
 
 See also: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-sap-connector/[SAP Connector in Anypoint Exchange]
 
-*Note:* This connector version is not supported for use in Design Center.
+== Audience
+
+* Starting user -- Read this Get Started document. Configure your Mule application in 
```

Comment:
Suggest using this format:

* Beginner user
   Read this ....

* Experienced user
   Read....

Reply:
Is this really a Get Started document, or is this a section? 
And can we word this so that it does not refer to itself?